### ISSUE TITLE
[Wave Collect][Xero] Fix the Tags menu item not showing in the LHN

### ISSUE THREAD

**Issue by lakchote** [Expensify/App#41831]
**Problem**

Tags menu item are not showing in the LHN

**Solution**

Let's implement it so they show in the LHN when customers or tracking categories  are imported as tags

The end result should look like this below:

<img width="1840" alt="image" src="https://github.com/Expensify/App/assets/21549771/eb527baf-d85b-4c90-9506-a04b64570855">

**Comment by lakchote** [comment]
> Just a heads up on that mock that it's got some outdated UI. Please use the page that exists in product already for tags.

Thank you Tom. I've updated the mock image.

**Comment by cdOut** [comment]
Hi, I’m Tymoteusz from Software Mansion, the expert agency, and I’d like to work on this task.

**Comment by mananjadhav** [comment]
I can help with the review once this is ready.

**Comment by lakchote** [comment]
@cdOut how is it going, do you need anything? If you hit any blockers don't hesitate to ping me. Thanks!

**Comment by mananjadhav** [comment]
@lakchote Can you assign this to me for the review so that I can track this.

**Comment by cdOut** [comment]
Hi, I'm going to put a PR for this tonight or early tomorrow, I'm currently working on finishing it up.

**Comment by lakchote** [comment]
Tags menu is showing on the LHN ([Slack](https://expensify.slack.com/archives/C036QM0SLJK/p1715695392769269?thread_ts=1715336564.073279&cid=C036QM0SLJK)). We need to wait for https://github.com/Expensify/App/pull/41562 to be merged and deployed to staging though, so everyone can test it.

@adelekennedy no payment due on this one, as @cdOut didn't have to put up a PR for it, it seems.

Closing the issue in the meantime.

**Comment by mananjadhav** [comment]
@lakchote Do we need this issue?

**Comment by lakchote** [comment]
Closing in favor of https://github.com/Expensify/Expensify/issues/397238.
